SimpleMovies/SimpleMovies/Scenes/List/ListViewControllerCopy.swift

```diff
@@ -0,0 +1,134 @@
+//
+//  ListViewControllerCopy.swift
+//  SimpleMovies
+//
+//  Created by André Sanches Bocato on 21/09/20.
+//  Copyright © 2020 Eduardo Sanches Bocato. All rights reserved.
+//
+
+import UIKit
+
+class ListViewControllerCopy: UIViewController {
+    
+    // MARK: - IBOutlets
+    
+    @IBOutlet weak var searchTextField: UITextField!
+    @IBOutlet weak var tableView: UITableView!
+    
+    // MARK: - Properties
+    
+    private var searchResults = [SearchResponse.Result]()
+    
+    // MARK: - Lifecycle
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupTableView()
+        FavoritesManager.shared.load()
```

Comment:
[NICE TO HAVE]
Evite colocar chamadas diretas no `viewDidLoad()`.  
Sugiro criar um método como `setup()` que encapsula todas as configurações iniciais da tela.